ui/apps/everest/src/components/db-actions/db-actions.tsx

```diff
@@ -163,6 +169,18 @@ export const DbActions = ({
             'aria-labelledby': 'row-actions-button',
           }}
         >
+          {showDetailsAction && pathname !== redirectURL && (
```

Comment:
why all of this ?
We have the "showDetailsAction" flag, no need for further checks in the component.
The DbActions is rendered in two places:
- DbClusterView, aka cluster list;
- Cluster details page.

We need to show the icon on the **cluster list**, not on the details page itself.
So, on that page, just pass the flag as true and remove this pathname check.